src/main/groovy/com/adarshr/gradle/testlogger/TestLoggerExtension.groovy

```diff
@@ -16,6 +17,7 @@ import static org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
 class TestLoggerExtension {
 
     ThemeType theme = STANDARD
+    LogLevel logLevel = LogLevel.LIFECYCLE
```

Comment:
We should also document this new property in the README under the relevant sections.